src/components/item/settings/GeolocationPicker.tsx

```diff
@@ -0,0 +1,153 @@
+import { ChangeEventHandler, useCallback, useEffect, useState } from 'react';
+
+import Clear from '@mui/icons-material/Clear';
+import {
+  Box,
+  IconButton,
+  LinearProgress,
+  List,
+  ListItemButton,
+  Stack,
+  TextField,
+  Tooltip,
+  Typography,
+} from '@mui/material';
+
+import { DiscriminatedItem } from '@graasp/sdk';
+
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
I think that `yarn add -D @types/leaflet-geosearch` should solve this.

```suggestion
```

Reply:
The package does not really exist! https://www.npmjs.com/package/@types/leaflet-geosearch but I'm not sure why I have this error I might say 🤷‍♀️

Reply:
Okay my bad 😅